public/styles/base/_utilities.scss

```diff
@@ -0,0 +1,7 @@
+.order-1 {
+  order: 1;
```

Comment:
we try to avoid using order to change the visual presentation from the source order for accessibility reasons. while it reorders text and a decorative icon in the current use case, the utility classes could be used in ways that negatively impact accessibility in the future. i lean towards checking the left/right position and updating the source accordingly to prevent the misuse of the utility class and set a good example for people who may come into our code base and look at what we're doing without the full context of the icons being decorative.